### DISCUSSION TITLE
Writing an article containing links to other articles

### DISCUSSION THREAD

**Discussion by m-shanti** [OrchardCMS/OrchardCore#11642] [Q&A]
Hi,
I'm writing an article (content type "Article") and I want refer to other articles in this article.
Of course I can create links
`[Some article](https://myweb.page/some-article)`
but to do that I have to know permalink of the article I want to refer. So I have to find the article in another tab, get its permalink and paste to my article. It's a bit uncomfortable

Is there any easy option to insert such links?

**Comment by jtkech** [comment]
We have the ContentPickerField for this, hmm but take care of circular relations when rendering, but normally an Article will render its related Articles in a summary way, so normally without rendering their own related ones ;)

**Reply by m-shanti** [reply]
Thanks @jtkech. ContentPickerField can be useful for index of references below article. My problem is how to quickly refer to items in the content of the article, e.g.

_[...] This framework supports [Domain-driven design](http://mywebpage.xy/ddd) and was invented by [John Smith](http://mywebpage.xy/john-smith) [...]_
